### DISCUSSION TITLE
Issue - toolbar dissapears, backdropClick not working on mobile

### DISCUSSION THREAD

**Discussion by iamitrovics** [fancyapps/ui#551] [General]
Hello,
it seems from time to time - depend where you click toolbar on mobile just dissapear. I guess when you click on content area or backdrop / outside content area. 

Also on all devices except mobile when you click outside content area e.g. backdrop it close the modal, but on mobile its not closing it. Even if i force it with  backdropClick : close .  Whatver options i try to force like any click it works on desktop but not on mobile. Something overrides it.

**Comment by fancyapps** [comment]
Hi,

There is a 'compact' mode that mimics the UX on mobile devices and can be disabled by setting 'compact:false'.

**Reply by iamitrovics** [reply]
@fancyapps  yeah that solved the issue. Maybe it should be little better explained in documentation :) Also i noticed on mobile when you are using data-slug  and for example open and close 5 images, then you need in browser to press 5 times back button in order to get back to previous page.

Update; its same on desktop as well even on your website https://fancyapps.com/fancybox/plugins/hash/
if you try to click in gallery few items in a row and then try to use back button it will not go back on previous page.
